tools/js-sdk-release-tools/package.json

```diff
@@ -23,7 +23,7 @@
   "license": "MIT",
   "dependencies": {
     "@azure-tools/openapi-tools-common": "^1.2.2",
-    "@ts-common/azure-js-dev-tools": "^22.4.22",
+    "@ts-common/azure-js-dev-tools": "^21.1.0",
```

Comment:
If this package isn't maintained any longer we should look to see if we can remove this dependency.

Reply:
sure, we will remove it